packages/babel-plugin-transform-regenerator/src/regenerator/visit.ts

```diff
@@ -1,13 +1,38 @@
 "use strict";
 
+import { types as t } from "@babel/core";
+import type { NodePath } from "@babel/traverse";
```

Comment:
Nitpick, but let's use a single import
```suggestion
import { types as t, type NodePath } from "@babel/core";
```